docs/source/installation.rst

```diff
@@ -39,20 +39,35 @@ There are fundamentally two different approaches to install ``IvadoMed``:
 Approach 1: Conda
 ------------------
 
-Step 1: Create new Conda Env called IvadoMedEnv (may taken 5 to 15 minutes)
```

Comment:
Sorry for being that guy but there's a typo here: _may taken_ should be _may take_